src/measurement.py

```diff
@@ -7,26 +18,22 @@ class Measurement:
     and are not drawn.
     """
```

Comment:
A measurement `may` define error bounds. Now it has to define error bounds, right?

Reply:
And they don't default to `0.0` anymore.